Source/player.cpp

```diff
@@ -19,41 +19,161 @@ int plr_sframe_size;
 int deathdelay;
 int plr_dframe_size;
 
+// address: 0x47F208
+//
+// ArmourChar maps from armor animation to letter used in graphic files.
+//
+// PSX ref (SLPS-01416): 0x8011B008
+// PSX def: char ArmourChar[4]
+// Note: char[3] is enough, though PSX details as char[4].
+// alias: player_armors
 const char ArmourChar[4] = { 'L', 'M', 'H', 0 };
+// address: 0x47F20C
+//
+// WepChar maps from weapon animation to letter used in graphic files.
+//
+// PSX ref (SLPS-01416): 0x80110B68
+// PSX def: char WepChar[10]
+// Note: char[9] is enough, though PSX details as char[10].
+// alias: player_weapons
 const char WepChar[10] = { 'N', 'U', 'S', 'D', 'B', 'A', 'M', 'H', 'T', 0 };

```

Comment:
4 is used since the loops continue until they hit a null.
`for (w = &WepChar[0]; *w; w++) {`

Reply:
Ah, that makes sense. Kind of which they did:

```cpp
for (i = 0; i < sizeof(WepChar)/sizeof(WepChar[0]); i++)
```